extensions/wikia/YouTube/YouTube.php

```diff
@@ -504,6 +488,7 @@ function embedGametrailers( $input, $argv, $parser ) {
 	}
 
 	if ( !empty( $gtid ) ) {
+		// www.gametrailers.com does not yet support HTTPS, to handle in (PLATFORM-3284)
 		$url = "http://www.gametrailers.com/remote_wrap.php?mid={$gtid}";
 		// return "<object type=\"application/x-shockwave-flash\" width=\"{$width}\" height=\"{$height}\"><param name=\"movie\" value=\"{$url}\"/></object>";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)

[project-id]: wikia:app